### PR TITLE
add asterisk for required field

### PR DESCRIPTION
**Pull request by ank1traj**
**[Issue](https://github.com/NikhilJoshua/thelazyentry/issues/4)** - Required fields Issue 

**Approach to solve** - Use CSS for styling and adding the asterisk before the required field

**Code Used** - `.required-field::before {
    content: "*";
    color: red;
    margin-left:2px
}`

**Issue level** - Easy

### CODE REVIEW

**Comment by NikhilJoshua** [comment]
@sroshan106 this might conflict the functionality. Please check the changes.

@ank1traj We are dynamically adding the asterisk(*) to empty field with class names. The changes you made might conflict with current functionality

File: index.html [hunk 1]
```diff
@@ -39,15 +39,15 @@ <h4>
...
 						<p class="labelStartTime">*</p>
 						<input type='time' name="startTime">
 						<button onclick="setStartTime()" title="Set to current time">Select Current Time</button>
 					</td>
 					
 					<td>
-						<label for="endTime" >End: </label>
-						<p class="labelEndTime">*</p>
+						<label class="required-field" for="endTime" >End: </label>
+						<p class="labelEndTime"></p>
```
Comment (NikhilJoshua) [inline index.html:50]: < p class="labelEndTime" >*< /p > * should be there I guess

File: index.html [hunk 2]
```diff
@@ -61,8 +61,8 @@ <h4>
 				</tr>
 				<tr class="fields-wrapper">
 					<td>
-						<label for="task"> Task: </label>
-						<p class="labelTask">*</p>
+						<label class="required-field" for="task"> Task: </label>
+						<p class="labelTask"></p>
```
Comment (NikhilJoshua) [inline index.html:65]: Please add * in between p tags

File: index.html [hunk 3]
```diff
@@ -25,7 +25,7 @@ <h4>
 			<table class="data-fields">
 				<tr class="fields-wrapper">
 					<td>
-						<label for="title">Title(Date): </label>
+						<label class="required-field"for="title">Title(Date): </label>
```
Comment (NikhilJoshua) [inline index.html:28]: Not a required field.

**Review by NikhilJoshua — CHANGES_REQUESTED** [review]

**Comment by NikhilJoshua** [comment]
@ank1traj I guess you can close this pull request now as the #4 issue is resolved

**Comment by ank1traj** [comment]
The issue has been already fixed. So, closing this PR

**Pull request closed without merging**